ir/module.go

```diff
@@ -93,10 +93,8 @@ func (m *Module) WriteTo(w io.Writer) (n int64, err error) {
 		fw.Fprintf("source_filename = %s\n", quote(m.SourceFilename))
 	}
 	// Data layout.
-	if len(m.DataLayout) > 0 {
-		// 'target' 'datalayout' '=' DataLayout=StringLit
-		fw.Fprintf("target datalayout = %s\n", quote(m.DataLayout))
-	}
+	// 'target' 'datalayout' '=' DataLayout=StringLit
+	fw.Fprintf("target datalayout = %s\n", quote(m.DataLayout))
```

Comment:
The output code seems unchanged. Only parsing code was updated in LLVM 11.0.

For instance, given this input LLVM IR code:

```llvm
target datalayout = ""

define i32 @main() {
	ret i32 42
}
```

Running `opt -S -o foo2 foo.ll` from LLVM 11.0 gives the following output:

```llvm
; ModuleID = 'foo.ll'
source_filename = "foo.ll"

define i32 @main() {
  ret i32 42
}
```

Notice that the `datalayout` target is omitted.

However, if we include a non-empty data layout, it is included with the same command.

Input:

```llvm
target datalayout = "e-p:32:32:32-i1:8:8-i8:8:8-i16:16:16-i32:32:32-i64:32:64-f32:32:32-f64:32:64-v64:64:64-v128:128:128-a0:0:64-f80:128:128"

define i32 @main() {
	ret i32 42
}
```

Output of `opt -S bar2.ll bar.ll`, using `opt` from LLVM 11.0:

```llvm
; ModuleID = 'foo.ll'
source_filename = "foo.ll"
target datalayout = "e-p:32:32:32-i1:8:8-i8:8:8-i16:16:16-i32:32:32-i64:32:64-f32:32:32-f64:32:64-v64:64:64-v128:128:128-a0:0:64-f80:128:128"

define i32 @main() {
  ret i32 42
}
```